R/calcTransportGDPshare.R

```diff
@@ -31,8 +31,8 @@ calcTransportGDPshare <- function() {
   mapGLO$RegionCode <- "GLO"
   w <- ggdc10[, , "VA_Q05.Summation of sector GDP"]
   share <- share %>%
-    toolFillWithRegionAvg(valueToReplace = 0, weight = w, regionmapping = mapH12) %>%
```

Comment:
@Renato-Rodrigues 

This silences the following warning. If you think we should rather address it, let me know.
```
~ WARNING: 
~ More than 50% missing values for: LAM, OAS, SSA, EUR, NEU, MEA, REF, CAZ
```

Reply:
This file can be removed altogether according to Renato